src/config/Config.zig

```diff
@@ -1336,6 +1336,11 @@ keybind: Keybinds = .{},
 ///   * `end` - Insert the new tab at the end of the tab list.
 @"window-new-tab-position": WindowNewTabPosition = .current,
 
+/// Whether tabs should always be displayed, regardless of how many tabs
+/// are open. If false (default), the tab bar is only displayed when there
+/// are two or more tabs.
+@"window-always-show-tabs": bool = false,
```

Comment:
This should note it's only GTK for now. This has been requested on macOS as well so I'm interested in making it work there eventually.